elliottlib/cli/find_bugs_cli.py

```diff
@@ -21,20 +21,27 @@
 @use_default_advisory_option
 @click.option("--mode",
               required=True,
-              type=click.Choice(['list', 'sweep', 'diff', 'qe']),
+              type=click.Choice(['list', 'sweep', 'diff', 'qe', 'blocker']),
               default='list',
               help='Mode to use to find bugs')
 @click.option("--status", 'status',
               multiple=True,
+              default=None,
               required=False,
-              default=['MODIFIED', 'VERIFIED', 'ON_QA'],
               type=click.Choice(constants.VALID_BUG_STATES),
-              help="Status of the bugs")
+              help="Include bugs of this status")
+@click.option("--exclude-status", 'exclude_status',
```

Comment:
This was added so we can use blocker in prepare-release job with `--exclude-status ON_QA`